lib/cfa/augeas_parser.rb

```diff
@@ -119,12 +155,20 @@ def []=(key, value)
       end
     end
 
+    # @param matcher [Matcher]
+    # @return [Array<AugeasElement>] matching elements
     def select(matcher)
       @data.select(&matcher)
     end
 
     # @note for internal usage only
     # @private
+    #
```

Comment:
No.
But an even "better" thing happens: it is excluded from the documentation, even if I run `yardoc --private` (which does show regular `private` methods). WTF.
So I change this to `@api private`